### DISCUSSION TITLE
Personal universe not indexing

### DISCUSSION THREAD

**Discussion by TuQmano** [r-universe-org/help#200] [General]
For some reason I have no idea about, my personal r-universe site (which seems to be ok) and all of its content is not searchable

**This is https://tuqmano.r-universe.dev/**
![image](https://user-images.githubusercontent.com/12114624/189491913-8d0126df-ab23-4025-954c-9cf23efc061b.png)

**When I search for my surname** `(https://r-universe.dev/search/#%22Ruiz%20Nicolini%22)`, for example, I get only my organization packages

![image](https://user-images.githubusercontent.com/12114624/189491979-19d73815-1db1-4821-be71-97119f8a9c78.png)

**Comment by jeroen** [comment]
Right, the reason is that by default we only show search results of packages that are "self-owned", i.e. they are in a registry that is owned by the same GitHub user that owns the package. This is because we had a lot of duplicates and forks showing up in search results, where a package is in multiple different universes, some of them not owned by the author. That was undesired.

In your case your universe registry is `tuqmano` but the packages live in other orgs, such as `politicaargentina` and `dnme-minturdep` so we can't verify that you are the owner, so we the system does not show them in search results now.

I am trying to think of a solution for this problem, such that we can relax this restriction a bit.

**Comment by jeroen** [comment]
OK I have updated the algorithm such that also packages in `tuqmano` that are maintained by user `tuqmano` show up, so you should see a few more now.

**Comment by TuQmano** [comment]
Thanks for the quick answer @jeroen  !

I just generate an organization r-universe for `politicaargentina` in order to test these differences. 

With your past update I was able to get more results in my search (my full name, for example), but not with my handle (`TuQmano`), 

Thanks again for all of this great work!
